kasa/tests/fakeprotocol_smart.py

```diff
@@ -80,6 +80,22 @@ def credentials_hash(self):
             "firmware",
             {"enable": True, "random_range": 120, "time": 180},
         ),
+        "get_alarm_configure": (
+            "alarm",
+            {
+                "get_alarm_configure": {
+                    "duration": 10,
+                    "type": "Doorbell Ring 2",
+                    "volume": "low",
+                }
+            },
+        ),
+        "get_support_alarm_type_list": ("alarm", {
+            "alarm_type_list": [
+                "Doorbell Ring 1",
+            ]
+        }),
+        "get_device_usage": ("device", {}),
```

Comment:
The out-of-the-box fw didn't apparently respond to this, but the currently installed one returns just `{}`.